packages/profiles/source/signed-transaction.dto.ts

```diff
@@ -131,11 +148,25 @@ export class ExtendedSignedTransactionData {
 	}
 
 	public total(): number {
+		if (this.isReturn()) {
+			return this.amount() - this.fee();
+		}
+
 		if (this.isSent()) {
 			return this.amount() + this.fee();
 		}
 
-		return this.amount();
+		let total = this.amount();
+
+		if (this.isMultiPayment()) {
+			for (const recipient of this.recipients()) {
+				if (recipient.address !== this.wallet().address()) {
+					total -= recipient.amount;
+				}
+			}
+		}
+
```

Comment:
Added `total` calculation for multipayment signed transactions. The same as confirmed transaction: https://github.com/ArdentHQ/platform-sdk/blob/d5b83462bf14c6ae1eb4556ffdfbf1e155999cb4/packages/profiles/source/transaction.dto.ts#L274